src/Client.php

```diff
@@ -968,6 +968,48 @@ public function generateDeploymentConfig(string $name, string $image_stream_tag,
     return $deploymentConfig;
   }
 
+  public function addProbeConfig(&$deployment_config, $probes) {
```

Comment:
Missing method doc - which belongs in the interface... along with the method declaration there ;)